mappings/1.8.8#1.8.9.tinydiff

```diff
@@ -1,25 +1,25 @@
 tiny	2	0
 c	net/minecraft/unmapped/C_0309511
-	f	Z	f_2867010		f_2867010
-	f	Lnet/minecraft/unmapped/C_0786431;	f_6447142		f_6447142
-	m	()Z	m_1993954		m_1993954
+	f	Z	f_2867010		disableRealmsNotifications
```

Comment:
This field is only `false` while the Realms bridge has not yet been invoked to create the notifications screen (which might fail if the Realms module is missing). The name should convey that.